graph/types/schema.graphqls

```diff
@@ -30,9 +30,35 @@ input AddDiscussionParticipantInput {
   isAnonymous: Boolean!
 }
 
+# TODO: implement
+input MediaInput {
+  type: String!,
+  mediaID: ID!
+}
+
+# TODO: implement
+input PollInput {
+  pollText: String!,
+  duration: Time!,
```

Comment:
Should duration be a measurement rather than a Time? Or should we mark `endTime` so it's an actual time here?